docs/src/main/asciidoc/kogito-pmml.adoc

```diff
@@ -0,0 +1,255 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Quarkus - Using Kogito to add prediction capabilities to an application
+
+include::./attributes.adoc[]
+:extension-status: preview
+
+This guide demonstrates how your Quarkus application can use Kogito to add business automation
+to power it up with predictions.
+
+Kogito is a next generation business automation toolkit that originates from well known Open Source projects
+Drools (for predictions). Kogito aims at providing another approach
```

Comment:
Forgot this one: you say `projects` plural but there is only one. In there you have two: https://github.com/quarkusio/quarkus/pull/19027/files#diff-0decdce40b1bf462f86568d4e3faf2060f23d9adc16503100b408e0c44682c54R14 .

Reply:
@gitgabrio have you seen this one?

Reply:
@gsmet 
Sorry - I do not get that: "projects" (plural) refers to the "Open Source projects" Kogito originates from (and they are more then just one)

Reply:
`that originates from well known Open Source projects Drools (for predictions)`
vs
`that originates from well known Open Source projects Drools (for business rules) and jBPM (for business processes)`

Either you cite several projects as done in the other guide or you need to use `such as Drools for predictions`. Otherwise it's not really correct (sorry for the nitpicking but we try to get our docs in as good shape as possible).

Reply:
@gmset

Thanks, now I see that! :+1: